sysconfigs/daint/packages.yaml

```diff
@@ -35,7 +35,7 @@ packages:
       variants: +cuda cuda_arch=60
       compiler: [gcc]
     cosmo-dycore:
-      variants: slave=daint +cuda cuda_arch=60 data_path=/scratch/snx3000/wbenjami/cosmo-serialized-data/
+      variants: slave=daint +cuda cuda_arch=60 data_path=/scratch/snx3000/wbenjami/cosmo-serialized-data/ slurm_args= "'--time=00:05:00 -C gpu -p normal -A g110 -N {0}'"
```

Comment:
I think --time should be part of the package tests (instead of spack). It is up to the package to decide how expensive they are, and this can actually change with different versions of the package, isnt?

Reply:
yes of course, I removed them, although it is not that important because the user can still change it later, it is just setting the default for Daint, so it was more of a suggestion.

Reply:
ah ok, yes, you are right